guide/12-enrich-data-with-thematic-information/part3_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>with and <em>instantiating </em>an</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>